pkg/apis/core/v1/storagecluster.go

```diff
@@ -309,6 +309,17 @@ type RollingUpdateStorageCluster struct {
 	// Defaults to 0 (pod will be considered available as soon as it is ready)
 	// +optional
 	MinReadySeconds int32 `json:"minReadySeconds,omitempty"`
+
+	// The default behavior is non-disruptive upgrades. This setting disables the default
+	// non-disruptive upgrades and reverts to the previous behavior of upgrading nodes in
+	// parallel without worrying about disruption.
+	Disruption DisruptionSpec `json:"disruption,omitempty"`
+}
+
+// DisruptionSpec contains disruption details
+type DisruptionSpec struct {
```

Comment:
let's change this to Disruption instead of DisruptionSpec as we just have single field allow for that

Reply:
Done @nikita-bhatia